src/mobile/package.json

```diff
@@ -19,7 +19,7 @@
     "android:dev": "./node_modules/.bin/cross-env NODE_ENV=development ./node_modules/.bin/react-native run-android",
     "ios": "./node_modules/.bin/react-native run-ios",
     "ios:dev": "./node_modules/.bin/cross-env NODE_ENV=development ./node_modules/.bin/react-native run-ios",
-    "test": "NODE_ENV=test ./node_modules/.bin/jest --forceExit",
+    "test": "REALM_DISABLE_ANALYTICS=true NODE_ENV=test ./node_modules/.bin/jest --forceExit",
```

Comment:
This will prevent Realm from sending analytics using `request`, which doesn't seem to cooperate with Jest. If Realm tries to send analytics, Jest will hang (see https://buildkite.com/iota-foundation/trinity-mobile-prs/builds/1198)